authenticity-collector/src/main.ts

```diff
@@ -22,7 +22,9 @@ async function bootstrap() {
 		.build();
 
 	const document = SwaggerModule.createDocument(app, config, options);
-	SwaggerModule.setup('/api/v1/authenticity-collector-docs', app, document);
+	SwaggerModule.setup('/docs', app, document);
+	//url for traefik
+	SwaggerModule.setup('/authenticity-docs', app, document);
```

Comment:
the url in traefik shall not come from the service itself but traefik shall route this route from `authenticity-collector/docs/` to the `/docs` of the authenticity collector.

Same for below